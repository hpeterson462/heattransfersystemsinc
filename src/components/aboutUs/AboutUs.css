.about-p {
  margin: auto;
  width: 80%;
  line-height: 2rem;
}

.profile-container {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  margin-top: 20px;
  padding: 20px;
  transform: translateY(20vh);
  transition: opacity 0.6s ease-out, transform 1.2s ease-out;
  background-color: #003366;
  z-index: -10;
}

.profile-container.is-visible {
  opacity: 1;
  transform: none;
  visibility: visible;
}

.profile-card {
  padding: 20px;
  width: 300px;
  text-align: center;
  font-family: 'Noto Sans', sans-serif;
}

.profile-card-inside {
  padding: 10px;
  border-radius: 2px;
  background-color: white;
}

.profile-img {
  width: 150px;
  border-radius: 50%;
}
