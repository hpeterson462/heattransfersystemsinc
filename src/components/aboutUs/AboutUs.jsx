import React, { useEffect } from 'react';
import steve from '../../assets/steve.jpg';
import daniel from '../../assets/daniel.jpg';
import bailey from '../../assets/bailey.jpg';
import paul from '../../assets/paul.jpg';
import "aos/dist/aos.css";
import AOS from "aos";
import './AboutUs.css';

export default function AboutUs() {

  useEffect(() => {
    AOS.init();
    AOS.refresh();
  }, []);

  return (
    <section>
      <h1 id="aboutus">
        About Us
      </h1>
      <h2>
        Our team has been involved in the sales, service, and engineering  of combustion systems and equipment for over 50 years that includes three generations. We have an extensive background in designing systems for many different industries.
      </h2>

      <div className="profile-container" data-aos="fade-zoom-in" data-aos-offset="200" data-aos-easing="ease-in-sine" data-aos-duration="900">

        <div className="profile-card">
          <img
            className="profile-img"
            src={steve}
            alt="Heat Transfer Systems employee Steve"
          />
          <div>
            <p>Steve Wirth</p>
            <p>President</p>
            <a href="mailto:steve@heattransfersystems.com" className="contact-email">
              steve@heattransfersystems.com
            </a>
          </div>
        </div>

        <div className="profile-card">
          <img
            className="profile-img"
            src={daniel}
            alt="Heat Transfer Systems employee Daniel"
          />
          <div>
            <p>Daniel Candaux</p>
            <p>Project & Services Manager</p>
            <a href="mailto:daniel@heattransfersystems.com" className="contact-email">
              daniel@heattransfersystems.com
            </a>
          </div>
        </div>

        <div className="profile-card">
          <img
            className="profile-img"
            src={bailey}
            alt="Heat Transfer Systems employee Bailey"
          />
          <div>
            <p>Bailey Wirth</p>
            <p>Production & Warehouse Manager</p>
            <a href="mailto:bailey@heattransfersystems.com" className="contact-email">
              bailey@heattransfersystems.com
            </a>
          </div>
        </div>

      </div>

    </section >
  )
}
