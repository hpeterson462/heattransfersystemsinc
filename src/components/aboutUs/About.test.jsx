import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import AboutUs from './AboutUs';

describe('AboutUs component', () => {
  afterEach(() => cleanup());

  it('renders AboutUs component', () => {
    const { asFragment } = render(<MemoryRouter><AboutUs /></MemoryRouter>);

    expect(asFragment()).toMatchSnapshot();
  });
});
