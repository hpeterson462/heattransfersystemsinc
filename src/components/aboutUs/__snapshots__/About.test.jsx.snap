// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs component renders AboutUs component 1`] = `
<DocumentFragment>
  <section>
    <h1
      id="aboutus"
    >
      About Us
    </h1>
    <h2>
      Our team has been involved in the sales, service, and engineering  of combustion systems and equipment for over 50 years that includes three generations. We have an extensive background in designing systems for many different industries.
    </h2>
    <div
      class="profile-container is-visible"
    >
      <div
        class="profile-card"
      >
        <img
          alt="Heat Transfer Systems employee Steve"
          class="profile-img"
          src="steve.jpg"
        />
        <div>
          <p>
            Steve Wirth
          </p>
          <p>
            President
          </p>
          <a
            class="contact-email"
            href="mailto:steve@heattransfersystems.com"
          >
            steve@heattransfersystems.com
          </a>
        </div>
      </div>
      <div
        class="profile-card"
      >
        <img
          alt="Heat Transfer Systems employee Paul"
          class="profile-img"
          src="paul.jpg"
        />
        <div>
          <p>
            Paul Deliramich
          </p>
          <p>
            Inside Sales & Support Manager
          </p>
          <a
            class="contact-email"
            href="mailto:paul@heattransfersystems.com"
          >
            paul@heattransfersystems.com
          </a>
        </div>
      </div>
      <div
        class="profile-card"
      >
        <img
          alt="Heat Transfer Systems employee Daniel"
          class="profile-img"
          src="daniel.jpg"
        />
        <div>
          <p>
            Daniel Candaux
          </p>
          <p>
            Project & Services Manager
          </p>
          <a
            class="contact-email"
            href="mailto:daniel@heattransfersystems.com"
          >
            daniel@heattransfersystems.com
          </a>
        </div>
      </div>
      <div
        class="profile-card"
      >
        <img
          alt="Heat Transfer Systems employee Bailey"
          class="profile-img"
          src="bailey.jpg"
        />
        <div>
          <p>
            Bailey Wirth
          </p>
          <p>
            Production & Warehouse Manager
          </p>
          <a
            class="contact-email"
            href="mailto:bailey@heattransfersystems.com"
          >
            bailey@heattransfersystems.com
          </a>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
